shopinvader/services/partner_mixin.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2019 Camptocamp (http://www.camptocamp.com).
+# @author Simone Orsi <simone.orsi@camptocamp.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# pylint: disable=method-required-super, consider-merging-classes-inherited
+
+from odoo import _
+from odoo.addons.component.core import AbstractComponent
+
+
+class PartnerServiceMixin(AbstractComponent):
+    _name = "shopinvader.partner.service.mixin"
+
+    def _post_create(self, partner):
```

Comment:
on update and create it would be nice to collect info on what has been created (eg: the address -> partner is notified)
or what has been updated (eg: diff of the fields to show in the notification)
if fields are tracked we can reuse mail tracking mechanism otherwise we can collect values before and after and inject the value in the ctx